backend/api/Database/Models/Source.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+#pragma warning disable CS8618
+namespace Api.Database.Models
+{
+    public class Source
+    {
+        [Key]
+        [Required]
+        [MaxLength(200)]
+        public string Id { get; set; }
+
+        [Required]
+        public string URL { get; set; }
+
```

Comment:
```suggestion
        [Required]
        public string sourceId { get; set; }

```

My thoughts have been that this URL field should point to either the echo mission or the custom mission. 

For echo mission that would be an ID: int (can be string)
For custom mission it would be a json file in blob storage: URL

If we call it `SourceId` we can have it as the int (as a string) for echo and the path or url to the json in blob storage for a custom mission.